code/modules/mining/mine_items.dm

```diff
@@ -280,17 +280,17 @@
 	item_state = "dpickaxe"
 	origin_tech = list(TECH_MATERIAL = 6, TECH_ENGINEERING = 4)
 	desc = "A pickaxe with a diamond pick head."
-	excavation_amount = 30
-
-	digspeed_unwielded = 20
+	excavation_amount = 50
+	autodrill = 1
```

Comment:
While a cool idea that I like, sort of defeats the idea of a pickaxe being a manual tool

Reply:
Reason was that until now the diamond pickaxe was really hard on peoples mouses, if you got out spamming the click for 20 minutes, your hand and your mouse would suffer. Did not really stop me from using it, but that was the feedback I got, so I figured the top end of pickaxes could do with this feature. This makes it so you can either bump through rocks without having to spam the mouse click, or you could use the mouse to quickly get rid of rocks in a wider area in front of you(a little like the double click on the resonator works right now), without having to click as much (old one was 30 on the amount, meaning you had to click at least 4 times per rock, it's been pretty much halved).